data/json/recipes/weapon/explosive.json

```diff
@@ -334,8 +329,8 @@
     "qualities": [ { "id": "SCREW", "level": 1 } ],
     "components": [
       [ [ "water", 1 ], [ "water_clean", 1 ], [ "salt_water", 1 ], [ "saline", 5 ] ],
-      [ [ "candy", 1 ], [ "cola", 1 ] ],
-      [ [ "vitamins", 10 ], [ "aspirin", 8 ] ],
+      [ [ "sugar", 16 ] ],
+      [ [ "chem_saltpetre", 24 ] ],
```

Comment:
This is probably a good change, but completely unrelated to the current PR, as worded.

Reply:
I'll add that to my description.